WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -314,6 +316,7 @@ class MediaLibraryViewController: UIViewController {
         service.deleteMedia(assets,
                             progress: updateProgress,
                             success: { [weak self] in
+                                WPAppAnalytics.track(.mediaLibraryDeletedItems, withProperties: ["number_photos_deleted": deletedItemsCount], with: self?.blog)
```

Comment:
Same as before, this should be `"number_of_items_deleted"` since we can also delete videos here.